.github/workflows/publish.yml

```diff
@@ -30,18 +30,26 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+
+      - uses: Swatinem/rust-cache@v2
+
       - name: Install mdbook
         run: cargo install mdbook
```

Comment:
You should probably pin these to a specific version rather than just whatever happens to be the latest, and add the versions to the cache key.

Reply:
The `@v2` part pins it to that release. But perhaps you're talking about the `cargo install` commands? Yeah, I agree and I'll move them over to use a version number next.

Reply:
Yes, let me put up a PR for that next.